src/components/common/ProjectNameSelect.tsx

```diff
@@ -64,7 +64,9 @@ export const useProjectNameSelectOptions = (defaultProject: string): TypeaheadSe
 
   return projects.length === 0 || !projectsLoaded || projectsLoadError
```

Comment:
Please change the nested ternarary

Reply:
I've pull the new ternary I've added out into it's own constant, so hopefully that looks a little better now. Will push that change shortly.